acceptance: TestDockerCLI failed

[(acceptance).TestDockerCLI failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2567865&tab=buildLog) on [release-20.2@38fe95db9438ebb4f90c0b3b75ea6492399ca1fe](https://github.com/cockroachdb/cockroach/commits/38fe95db9438ebb4f90c0b3b75ea6492399ca1fe):

```
=== RUN   TestDockerCLI
--- FAIL: TestDockerCLI (389.82s)
    test_log_scope.go:154: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestDockerCLI883624499
    test_log_scope.go:63: use -show-logs to present logs inline
=== RUN   TestDockerCLI/test_demo_global.tcl
    --- FAIL: TestDockerCLI/test_demo_global.tcl (91.78s)
        cli_test.go:89: non-zero exit code: 1

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestDockerCLI PKG=./pkg/acceptance TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #58171 acceptance: TestDockerCLI failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestDockerCLI.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cli/interactive_tests/test_demo_global.tcl
+++ b/pkg/cli/interactive_tests/test_demo_global.tcl
@@ -1,7 +1,9 @@
 #! /usr/bin/env expect -f
 
-# This testcase is disabled until #56264 is resolved
 source [file join [file dirname $argv0] common.tcl]
+
+# Set a larger timeout since we are going global.
+set timeout 90
 
 start_test "Check --global flag runs as expected"
 

